components/FA2Transfer.tsx

```diff
@@ -0,0 +1,271 @@
+      fetch(
+        // `${API_URL}/v1/tokens/balances?account=${state.currentContract}&offset=${offset}&limit=20&token.metadata.name.as=*${value}*`
+        `${API_URL}/v1/tokens/balances?account=tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb&offset=${offset}&limit=20&token.metadata.name.as=*${value}*`
+      )
+        .catch(e => {
+          console.log(e);
+          return {
+            json() {
+              return Promise.resolve([]);
+            },
+          };
+        })
+        .then(res => res.json())
+        .then((v: fa2Token[]) => {
+          setCanSeeMore(v.length === FETCH_COUNT);
+
+          return Promise.resolve(v);
+        }),
+    [state.currentContract]
+  );
+
+  useEffect(() => {
+    const value = getFieldProps(makeName("token")).value as fa2Token | "";
+
+    if (!value) return;
+
+    updateValues(value);
+  }, []);
+
+  useEffect(() => {
+    setIsFetching(true);
+
+    debounce(
+      () =>
+        fetchTokens(filterValue, 0).then((v: fa2Token[]) => {
+          setOptions(v.map(tokenToOption));
+          setIsFetching(false);
+        }),
+      150
+    );
+  }, [fetchTokens, filterValue]);
+
+  return (
+    <div className="flex flex-col space-y-4 md:flex-row md:space-y-0 md:space-x-4">
+      <div className="w-full md:grow">
+        <Field name={makeName("token")}>
+          {() => (
+            <Select
+              placeholder="Please select an FA2 token"
+              className={!currentToken ? "md:mt-7" : ""}
+              label=""
+              withSeeMore={canSeeMore}
+              onSeeMore={() => {
+                fetchOffsetRef.current += 20;
+
+                fetchTokens(filterValue, fetchOffsetRef.current).then(
+                  (v: fa2Token[]) => {
+                    setOptions(current => current.concat(v.map(tokenToOption)));
+                  }
+                );
+              }}
+              onSearch={setFilterValue}
+              onChange={newValue => {
+                updateValues(newValue.token);
+              }}
+              value={!!currentToken ? tokenToOption(currentToken) : undefined}
+              options={options}
+              loading={isFetching}
+              renderOption={({
+                thumbnailHash,
+                tokenId,
+                contractAddress,
+                label,
+              }) => {
+                return (
+                  <div className="flex">
+                    <div className="w-1/6 overflow-hidden rounded bg-zinc-500/50">
+                      {!!thumbnailHash ? (
+                        <img
+                          src={`${THUMBNAIL_URL}/${thumbnailHash}`}
+                          alt={label}
+                          className="h-auto w-full"
+                        />
+                      ) : (
+                        <img
+                          src={
+                            "https://uploads-ssl.webflow.com/616ab4741d375d1642c19027/61793ee65c891c190fcaa1d0_Vector(1).png"
```

Comment:
I'm thinking
1. can the logo apply the mask like the one in "color" section in the 2nd column and 2nd row?
https://tailwind-elements.com/docs/standard/content-styles/mask/
2. what remove background color?

Reply:
Current:
![image](https://github.com/marigold-dev/tzsafe-ui/assets/115862/87642652-a7fb-4152-a78a-1ed6d6bf6d49)

I'd like it:
![image](https://github.com/marigold-dev/tzsafe-ui/assets/115862/b3661e50-e947-4625-bfc1-690b81dd5120)